qt/MakeDialog.cc

```diff
@@ -35,10 +35,10 @@ class MakeProgressDialog : public BaseDialog
     MakeProgressDialog(Session& session, tr_metainfo_builder& builder, QWidget* parent = nullptr);
 
 private slots:
-    void onButtonBoxClicked(QAbstractButton*);
+    void onButtonBoxClicked(QAbstractButton* button);
     void onProgress();
 
-private:
+private /* NOLINT(readability-redundant-access-specifiers) */ :
```

Comment:
I don't personally have any issue with "redundant" access specifiers; it seems perfectly normal to me to e.g. separate methods from variables. And then there's this here, which is clearly not redundant from human and Qt POV despite `slots` being expanded to nothing, but requires a comment...

Reply:
Reverted 4a3fdafa9263e76dd236aeff8526b78d5fd2e3b0